deb/openmediavault-clamav/usr/share/openmediavault/engined/module/clamav.inc

```diff
@@ -25,6 +25,10 @@ class ClamAV extends \OMV\Engine\Module\ServiceAbstract implements
 		return "clamav";
 	}
 
+	public function getDescription() {
+		return gettext("ClamAV® is an open-source antivirus engine for detecting trojans, viruses, malware & other malicious threats.");
```

Comment:
Sounds good, but maybe a little bit too long (see [function documentation](Get a brief description)). What about `Anti-virus utility for Unix`? Found at https://packages.debian.org/en/bullseye/clamav-daemon.